### DISCUSSION TITLE
Recommended way to backup UTM setup (and VMs) ?

### DISCUSSION THREAD

**Discussion by securixhk** [utmapp/UTM#5234] [UTM Help (macOS)] [answered]
Hello,

Ive been through the whole documentation and I haven't seen _(at least, I think :) )_ any guide/recommendations about practices for saving the whole UTM setup/configuration.

I wonder if it's sufficient to backup the whole **`~Library/Containers/UTM`** folder ?

A simple folder restoration is sufficient ?

Thanks in advance.

**Comment by osy** [comment, marked as answer]
Yes that should work

**Reply by securixhk** [reply]
Thanks a lot @osy for the confirmation :) !

**Reply by theabhinavdas** [reply]
@osy sorry, does this work across host architectures? for example, if i were to move from an arm host to an x86_64 host?

**Reply by fhsinchy** [reply]
@theabhinavdas I don't think that'd work since the guest OSes matches the host architecture.

**Reply by theabhinavdas** [reply]
Gotcha. Thanks, @fhsinchy! :)
